dGame/dComponents/TriggerComponent.cpp

```diff
@@ -113,22 +160,169 @@ std::vector<Entity*> TriggerComponent::GatherTargets(LUTriggers::Command* comman
 
 	if (command->target == "self") entities.push_back(m_Parent);
 	else if (command->target == "zone") { /*TODO*/ }
-	else if (command->target == "target") { /*TODO*/ }
-	else if (command->target == "targetTeam") { /*TODO*/ }
-	else if (command->target == "objGroup") entities = EntityManager::Instance()->GetEntitiesInGroup(command->targetName);
-	else if (command->target == "allPlayers") { /*TODO*/ }
-	else if (command->target == "allNPCs") { /*TODO*/ }
-
-	if (optionalTarget) entities.push_back(optionalTarget);
+	else if (command->target == "target" && optionalTarget) entities.push_back(optionalTarget);
+	else if (command->target == "targetTeam" && optionalTarget) {
+		auto* team = TeamManager::Instance()->GetTeam(optionalTarget->GetObjectID());
+		for (const auto memberId : team->members) {
+			auto* member = EntityManager::Instance()->GetEntity(memberId);
+			if (member) entities.push_back(member);
+		}
+	} else if (command->target == "objGroup") entities = EntityManager::Instance()->GetEntitiesInGroup(command->targetName);
+	else if (command->target == "allPlayers") {
+		for (auto* player : Player::GetAllPlayers()) {
+			entities.push_back(player);
+		}
+	} else if (command->target == "allNPCs") { /*UNUSED*/ }
```

Comment:
Unused in the live client or unused in general?

Reply:
In general, through all the files that we have, I have not seen any use of it

Reply:
It's still referenced in he client itself, but for as if it was ever used, looks like no